docs/guides/ultimate-guides/torrents/ultimate-intro-to-torrents/index.md

```diff
@@ -0,0 +1,152 @@
+---
+slug: ultimate-intro-to-torrents
+title: The Ultimate Intro to Torrents
+tags: [torrent, tutorial]
+authors: TechSquidTV
+---
+
+# What is a Torrent?
+
+More than likely, you are at least somewhat familiar with torrents. They are a way to download files from the internet.
+But how do they work? What are the benefits of using torrents? How do you use them? Are they safe? Consider the following your guide to _torrents_.
+
+## The Background of BitTorrent
+
+In the late 90s [Bram Cohen](https://en.wikipedia.org/wiki/Bram_Cohen) ([twitter](https://twitter.com/bramcohen)), a software developer, was working for a company called [MojoNation](<https://en.wikipedia.org/wiki/Mnet_(peer-to-peer_network)#MojoNation>) that was trying to create an encrypted decentralized file storage system for commercial-use.
+Unfortunately, the company was faced with severe financial difficulties and would close in 2002.
+Just before then, Cohen would realize that the technology he had been working on, while not a commercial success, could be used to create a new file sharing protocol that could dramatically speed up the process of sharing large files, such as [porn collections](https://www.wired.com/2005/01/bittorrent-2/).
+Other peer-to-peer file sharing protocols at the time existed, such as ["Kazaa"](https://en.wikipedia.org/wiki/Kazaa), but they all were dependent on downloading complete files from a single user or "peer".
```

Comment:
[Recommendation] I'm not quite sure if content creators are too familiar with Kazaa. I think another example worth mentioning would be [Napster](https://www.lifewire.com/history-of-napster-2438592#:~:text=As%20of%202022%2C%20Napster%20continues,Mint%20Songs%2C%20a%20Web3%20startup.). It also influenced content-sharing. 
```suggestion
Other peer-to-peer file sharing protocols at the time existed, such as ["Kazaa"](https://en.wikipedia.org/wiki/Kazaa), but they all were dependent on downloading complete files from a single user or "peer".
```

Reply:
I believe Napster came out a littler later than Kazaa. Though more popular, I think this is the right answer for the time period.